functions.php

```diff
@@ -161,7 +181,7 @@ function hello_elementor_content_width() {
 	 */
 	function hello_elementor_check_hide_title( $val ) {
 		if ( defined( 'ELEMENTOR_VERSION' ) ) {
-			$current_doc = \Elementor\Plugin::instance()->documents->get( get_the_ID() );
+			$current_doc = Plugin::instance()->documents->get( get_the_ID() );
```

Comment:
Also this, apologies, revert together with the `use` statement above

Reply:
Reverted.